src/Command/MakeUserAccountCommand.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Command;
+
+use App\Service\DogCRUDService;
+use App\Entity\Dog;
+use Symfony\Component\PasswordHasher\Hasher\UserPasswordHasherInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(
+    name: 'app:make-user-account',
```

Comment:
Are we using PHP 8? When I try to run `make migrate` symfony complains about the extra `,`:
```
ParseError {#411
  #message: "syntax error, unexpected ','"
  #code: 0
  #file: "./src/Command/MakeUserAccountCommand.php"
  #line: 16
  trace: {
    ./src/Command/MakeUserAccountCommand.php:16 {
      Composer\Autoload\includeFile($file)^
      › #[AsCommand(
      ›     name: 'app:make-user-account',
      ›     description: 'Create a user account for testing purposes',
    }
    ./vendor/composer/ClassLoader.php:428 { …}
    ./vendor/symfony/error-handler/DebugClassLoader.php:288 { …}
    Symfony\Component\ErrorHandler\DebugClassLoader->loadClass() {}
    spl_autoload_call() {}
    ./vendor/symfony/config/Resource/ClassExistenceResource.php:78 { …}
    ./vendor/symfony/dependency-injection/ContainerBuilder.php:370 { …}
    ./vendor/symfony/dependency-injection/Loader/FileLoader.php:217 { …}
    ./vendor/symfony/dependency-injection/Loader/FileLoader.php:105 { …}
    ./vendor/symfony/dependency-injection/Loader/YamlFileLoader.php:699 { …}
    ./vendor/symfony/dependency-injection/Loader/YamlFileLoader.php:256 { …}
    ./vendor/symfony/dependency-injection/Loader/YamlFileLoader.php:176 { …}
    ./vendor/symfony/dependency-injection/Loader/YamlFileLoader.php:132 { …}
    ./vendor/symfony/config/Loader/FileLoader.php:159 { …}
    ./vendor/symfony/config/Loader/FileLoader.php:98 { …}
    ./vendor/symfony/dependency-injection/Loader/FileLoader.php:66 { …}
    ./vendor/symfony/dependency-injection/Loader/Configurator/ContainerConfigurator.php:64 { …}
    ./vendor/symfony/framework-bundle/Kernel/MicroKernelTrait.php:58 { …}
    ./vendor/symfony/framework-bundle/Kernel/MicroKernelTrait.php:188 { …}
    ./vendor/symfony/dependency-injection/Loader/ClosureLoader.php:39 { …}
    ./vendor/symfony/config/Loader/DelegatingLoader.php:40 { …}
    ./vendor/symfony/framework-bundle/Kernel/MicroKernelTrait.php:196 { …}
    ./vendor/symfony/http-kernel/Kernel.php:651 { …}
    ./vendor/symfony/http-kernel/Kernel.php:547 { …}
    ./vendor/symfony/http-kernel/Kernel.php:789 { …}
    ./vendor/symfony/http-kernel/Kernel.php:128 { …}
    ./vendor/symfony/framework-bundle/Console/Application.php:168 { …}
    ./vendor/symfony/framework-bundle/Console/Application.php:74 { …}
    ./vendor/symfony/console/Application.php:171 { …}
    ./vendor/symfony/runtime/Runner/Symfony/ConsoleApplicationRunner.php:54 { …}
    ./vendor/autoload_runtime.php:35 { …}
    ./bin/console:11 { …}
  }
}
```
I had to comment out the command to get this working...

Reply:
Ah my bad. I'm using 8. It seem to just pick attributes for me and not annotations for metadata for some reason. I'll fix it in the PR 👍